setup.py

```diff
@@ -19,7 +19,7 @@
     install_requires.append('greenlet')
 
 setup(name='neovim',
-      version='0.1.10',
+      version='0.1.11dev',
```

Comment:
setuptools_scm?
See https://github.com/pytest-dev/pytest-mock/pull/67/files for an example integration.